e2e/regression/testdata/pod-secret-and-cm/pod.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod
+  namespace: "@@REPLACE_NAMESPACE@@"
+spec:
+  runtimeClassName: contrast-cc
+  containers:
+    - name: test
+      image: ghcr.io/edgelesssys/bash@sha256:cabc70d68e38584052cff2c271748a0506b47069ebbd3d26096478524e9b270b
+      command: ["/usr/local/bin/bash", "-c"]
+      args:
+        - set -xeo pipefail; if [[ $CM_CONTENT != "foobar" ]]; then exit 1; fi; if [[ $SECRET_CONTENT != "barfoo" ]]; then exit 1; fi; sleep infinity
```

Comment:
It makes sense to test the content here, but can we tell when the pod is crashlooping? Maybe add a readiness probe that tests for a marker file or something.